synapse_antispam/mjolnir/antispam.py

```diff
@@ -162,7 +162,7 @@ def __init__(self, config, api):
     async def check_event_for_spam(
         self, event: "synapse.events.EventBase"
     ) -> Union[bool, str]:
-        return self.antispam.check_event_for_spam(event)
+        return self.antispam.check_event_for_spam(event) or self.message_max_length.check_event_for_spam(event)
```

Comment:
Nit: the antispam APIs are hard to read because it's never clear whether you should return `True` or `False` to indicate spam.

I *think* it would be clearer if you rewrote this block as

```python
if self.antispam.check_event_for_spam(event):
  # User/server/... banned.
  return True
if self.message_max_length.check_event_for_spam(event):
  # Message too long.
  return True
return False
```